src/coffee/stockimport.coffee

```diff
@@ -371,8 +371,20 @@ class StockImport
             .fetch()
             .then (result) =>
               @_updateInventory(entry, result.body, tryCount + 1)
+            .catch (err) =>
+              if (err.statusCode == 404)
+                debug "It seem that stock update has conflicted with parallel stock deletion process."
+                + "We cannot update it anymore and will create a new stock."
+                @client.inventoryEntries.create(entry)
+              else
+                debug "Error on handling 409 stock update error. Details: #{JSON.stringify(err)}"
```

Comment:
`Error 409 occurred on stock update`

Reply:
This log is only for the case when you got an error when handling 409 update. So it's not a 409 error.

Reply:
okay then... LGTM